### ISSUE TITLE
conda config command section includes invalid directory of .condarc files

### ISSUE THREAD

**Issue by Silverteal** [conda/conda#14331]
### Checklist

- [x] I added a descriptive title
- [x] I searched open reports and couldn't find a duplicate

### What happened?

https://docs.conda.io/projects/conda/en/stable/commands/config.html#config-file-location-selection
Paragraph "Config File Location Selection" seems referring to incorrect native directory of readthedocs.org.

### Additional Context

![Image](https://github.com/user-attachments/assets/e5c15051-0617-4481-86c0-fb28fe7108bd)

**Comment by travishathaway** [comment]
@Silverteal,

I agree with this. That example is not ideal. We would be happy to review pull requests that change this.

**Comment by prady0t** [comment]
It is due to 
https://github.com/conda/conda/blob/405fbaaa7c85e5a528d2a824158d9c1f3fa89846/conda/base/context.py#L123
We can replace it with :
```py
user_rc_path = "~/.condarc"
```
